app/jobs/digest_spam_job.rb

```diff
@@ -0,0 +1,15 @@
+class DigestSpamJob
+  include Sidekiq::Worker
+  def perform(frequency_digest)
```

Comment:
Later on we could maybe add a way for moderators to customize the frequency of their digest so that only those who want it will receive theirs.

Reply:
Right now it just checks for tags digest:daily, digest:weekly, and if any of them is present then digest is sent. rn _it tasks whatever is the settings of the moderator in the notification settings_ page. I will add more customization for moderators specifically for spam digest like frequency, type, and status (By tagging different tag like spam_digest:daily)
I was thinking of doing this on /spam2 and make the settings page there. But as of now, it is not merged. Should I add it to the current notification settings page for moderators??